locales/da.json

```diff
@@ -152,12 +154,12 @@
 	"This will update Distance settings for ALL tracked Raids & Eggs": "Dette vil opdatere afstands instillingen for ALLE sporede Raids og Æg",
 	"Update Distance for all tracked Quests?": "Skal afstanden opdateres for alle sporede Missioner?",
 	"This will update Distance settings for ALL tracked Quests": "Dette vil opdatere afstands instillinger for ALLE sporede Missioner",
-	"Successfully Updated Distance on ALL Pokémons": "Afstand er opdateret for ALLE Pokémons",
-	"Successfully Updated Distance on ALL Raids & Eggs": "Afstand er opdateret for ALLE  Raids & Æg",
-	"Successfully Updated Distance on ALL Quests": "Afstand er opdateret for ALLE Missioner",
```

Comment:
You shouldn't change those 3 lines (at least not the left part) or it will not translate anymore as the English text won't match the code anymore

Reply:
I don't know why i changed them. But I assumed it was becaused they didn't match what was in the french one. So i assume they were wrong in the current commit. I can check and make a new PR if im wrong.